src/pressure/VCycleMG.cpp

```diff
@@ -294,7 +297,9 @@ void VCycleMG::pressure(Field *out, Field *b, real t, bool sync) {
         auto bsize_i = boundary->getSize_innerList();
         size_t *d_iList = boundary->get_innerList_level_joined();
 
-        while (r > tol_res && act_cycles < max_cycles && relaxs < max_relaxs) {
+        while (r > tol_res &&
+                static_cast<int>(act_cycles) < max_cycles &&
```

Comment:
instead of a cast, just define act_cycles and relaxs as integer ?

Reply:
changed also the class internal variables levels, cycles and relaxs to int. I think tyhere is no point in using size_t if the input is already an int. Pls check. 17898e1